openslides/core/projector.py

```diff
@@ -1,40 +1,52 @@
-from typing import Generator, Type
+from typing import Any, Dict
 
-from ..utils.projector import ProjectorElement
-from .exceptions import ProjectorException
-from .models import Countdown, ProjectorMessage
+from ..utils.projector import register_projector_element
 
 
-class Clock(ProjectorElement):
-    """
-    Clock on the projector.
-    """
-    name = 'core/clock'
+# Important: All functions have to be prune. This means, that thay can only
+#            access the data, that they get as argument and do not have any
+#            side effects. They are called from an async context. So they have
+#            to be fast!
```

Comment:
Please readd 'core/clock' projector element.

Reply:
I added a todo. I wan't to talk about this with the client devs.